db/schema.rb

```diff
@@ -358,6 +371,28 @@
   add_index "samples", ["molecule_id"], name: "index_samples_on_sample_id", using: :btree
   add_index "samples", ["user_id"], name: "index_samples_on_user_id", using: :btree
 
+  create_table "scifinding_credentials", force: :cascade do |t|
```

Comment:
these tables are odd
`rake db:drop && rake db:create && rake db:migrate` and then updating schema.rb will help